api/taskcluster/webhook.go

```diff
@@ -115,27 +155,8 @@ func (b branchInfos) GetNames() []string {
 	return names
 }
 
-func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
+func processTaskclusterBuild(ctx context.Context, taskGroupID, taskID string, sha string, labels ...string) (bool, error) {
```

Comment:
Take an `aeAPI` here instead of `ctx`. All the callers have already created an `aeAPI` before calling this function.